front-end/src/util/__tests__/formatBnBalance.test.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import formatBnBalance from '../formatBnBalance';
+
+jest.mock('../getNetwork', () => jest.fn(() => {return 'polkadot';}));
+const getNetwork = require('../getNetwork');
+
+describe('Formatting BN Balances', () => {
+	beforeEach(() => {
+		jest.resetModules();
+	});
+
+	const formattedBalance1 = '1.00 ';
+	const formattedBalance2 = '1 000 ';
+	const formattedBalance3 = '53.0 KSM';
+	const formattedBalance4 = '53.0 DOT';
+
+	it('in Kusama', () => {
```

Comment:
What does it do? We should have a better description like `it('verifies xyz in Kusama')`

Reply:
I have another `describe` on line 10 (`Get Encoded Address`) -> so you get `Get Encoded Address in Kusama`